src/CommunityToolkit.Maui.UnitTests/CommunityToolkit.Maui.UnitTests.csproj

```diff
@@ -1,30 +1,120 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Let's remove all of these new empty lines

Reply:
VS got crazy during rebase